mission-control/docs/notifications/concepts/silences.mdx

```diff
@@ -2,16 +2,20 @@
 title: Silences
 ---
 
-Silences prevent notifications from being sent. 
-Silences are attached to a particular resource (`catalog`, `health check`, or `component`) and are active for a specified duration.
+A silence is a way to temporarily suppress notifications. Each silence has:
+
+- A set duration - like an expiration timer
+- A scope - it can be applied to either:
+  - Individual resources (a specific catalog, health check, or component)
+  - Multiple matching resources using filters
 
 :::note
 Notifications that aren't sent due to silence are still visible in the notification history for auditing purposes.
 :::
 
 ## Use cases
```

Comment:
Can we add examples with 
- Regex on name
- Namespace / name 
- Catalog traverse up e.g. to match labels on the namespace